app/browser/share.js

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const assert = require('assert')
+const tabState = require('../common/state/tabState')
+const {shell} = require('electron')
+const tabs = require('./tabs')
+
+const templateUrls = {
+  email: 'mailto:?subject={title}&body={url}',
+  facebook: 'https://www.facebook.com/sharer.php?u={url}',
+  pinterest: 'https://pinterest.com/pin/create/bookmarklet/?url={url}&description={title}',
+  twitter: 'https://twitter.com/intent/tweet?url={url}&text={title}&hashtags=brave',
+  googlePlus: 'https://plus.google.com/share?url={url}',
+  linkedIn: 'https://www.linkedin.com/shareArticle?url={url}&title={title}',
+  buffer: 'https://buffer.com/add?text={title}&url={url}',
+  digg: 'http://digg.com/submit?url={url}&title={title}',
```

Comment:
that's an oversight but we aren't using it because Brad asked not to, I'll change it in case we do use it in the future though.